src/editors/array.js

```diff
@@ -393,8 +395,8 @@ JSONEditor.defaults.editors.array = JSONEditor.AbstractEditor.extend({
       else if(this.value.length === 1) {      
         this.remove_all_rows_button.style.display = 'none';  
 
-        // If there are minItems items in the array, hide the delete button beneath the rows
-        if(minItems || this.hide_delete_buttons) {
+        // If there are minItems items in the array, or configured to hide the delete_last_row button, hide the delete button beneath the rows
+        if(minItems || this.hide_delete_buttons || this.hide_delete_last_row_buttons) {
```

Comment:
Can't you just check `if(minItems || this.hide_delete_last_row_buttons)`?  Also checking for `this.hide_delete_buttons` seems redundant.